clients/drcachesim/CMakeLists.txt

```diff
@@ -391,6 +391,13 @@ if (X86 AND X64 AND ZIP_FOUND)
   add_test(NAME tool.scheduler_launcher
     COMMAND scheduler_launcher -trace_dir ${trace_dir})
   set_tests_properties(tool.scheduler_launcher PROPERTIES TIMEOUT ${test_seconds})
+
+  set(trace_dir_2
+    "${PROJECT_SOURCE_DIR}/clients/drcachesim/tests/drmemtrace.schedtest.x64")
```

Comment:
Since the issue seems to be that there is an empty thread in here maybe we don't want to check in this test data but rather prevent future empty traces.  Maybe add an invariant check that a (non-filtered) trace has at least one instruction, too.